admin/src/pages/UserSearch.spec.js

```diff
@@ -199,14 +199,34 @@ describe('UserSearch', () => {
       })
     })
 
+    describe('change user role', () => {
+      const userId = 4
+
+      it('to admin', async () => {
+        const now = new Date()
+        await wrapper
+          .findComponent({ name: 'SearchUserTable' })
+          .vm.$emit('updateIsAdmin', userId, now)
+        expect(wrapper.vm.searchResult.find((obj) => obj.userId === userId).isAdmin).toEqual(now)
```

Comment:
```suggestion
        expect(wrapper.vm.searchResult.find((obj) => obj.userId === userId).isAdmin).toEqual(expect.any(Date))
```
The exact date is not important. It comes from the backend.

Reply:
Done